crates/router/src/core/payments/types.rs

```diff
@@ -178,6 +178,8 @@ impl MultipleCaptureData {
 
 #[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
 pub struct SurchargeDetails {
+    /// original_amount
+    pub original_amount: i64,
```

Comment:
Shall we rename this to payment amount instead of original_amount?

Reply:
There is another field in this struct called `final_amount`. The idea was `final_amount = original_amount + surcharge_amount + tax_on_surcharge_amount`. 

So final_amount is treated as payment_amount. 
Wouldn't changing original_amount to payment_amount be contradicting?